metrics/response_stats.go

```diff
@@ -81,9 +91,9 @@ type ResponseStats struct {
 func PublishResponseStats(commonStats *CommonStats, duration time.Duration, err error) bool {
 	response := createResponseStats(commonStats, duration, err)
 	if  strings.Compare(commonStats.PaymentMode,"free-call") ==0  {
-		Publish(response, config.GetString(config.FreeCallEndPoint),commonStats)
+		Publish(response, config.GetString(config.FreeCallEndPoint)+ "/usage/freecalls",commonStats)
```

Comment:
Where one can find metrics endpoint descriptions?

Reply:
We are looking at building a Swagger UI to do this for Market place service   , work in progress ..

Reply:
@anandrgitnirman , my main concern is that I cannot check that protocol (see comment https://github.com/singnet/snet-daemon/pull/423/files#r348327583) and endpoints are used properly if I don't have protocol description.